src/server.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+const app =express()
+const port = 5000
+
+app.get('/',(req,res)=>{
+    res.send('welcome to our project')
+})
+
+app.listen(port,()=>{
+    console.log(`app is running on ${port}`)
+})
```

Comment:
- [ ] I am going to merge to the develop branch then go ahead and deploy it on `render` or `cyclic`